src/Symfony/Component/Config/Tests/Util/XmlUtilsTest.php

```diff
@@ -169,6 +169,8 @@ public function getDataForPhpize(): array
             [1, '1'],
             [-1, '-1'],
             [0777, '0777'],
+            ['-0777', '-0777'],
```

Comment:
isn't this supposed to be parsed to `-0777` (aka no quotes)?

Reply:
Well, `decoct` has hard times with negative numbers:
```php
echo decoct(-1); // "1777777777777777777777"
```
This makes me wonder if we should support negative octal numbers. Some research on octal notation and negative numbers reinforced this idea.
If you think it worth to add a few lines to support it (like checking the strings starts with `-` and then multiply by -1 the final result, something like that), I can add it. But it seems a bit "hacky" :smile:

Reply:
On the other end:
```php
echo -0777; // -511
```

Reply:
You're right! I'll come up with a solution to handle this. 👍

Reply:
This should be better now!